src/adapters/scene.js

```diff
@@ -113,7 +113,11 @@ Scene.prototype.initMapBox = function() {
               offset: [(layout.sizes.panelWidth + layout.sizes.sideBarWidth) / 2, 0],
             });
           }
-          window.app.loadPoi(mapPoi, { disableMapPan: true });
+
+          const poi = await window.app.loadPoi(mapPoi, { disableMapPan: true });
+          if (poi) {
+            this.addMarker(poi);
+          }
```

Comment:
this change is probably obsolete and should have been absent after rebase, no ?

Reply:
Absolutely no idea.

Reply:
@amatissart is right, it's done through events in the `loadPoi` method.